### DISCUSSION TITLE
Delete white space from div tag

### DISCUSSION THREAD

**Discussion by MinalPatilRPA** [tailwindlabs/tailwindcss#11995] [Help] [answered]
Hello Team,

![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/a5a6665d-1959-46b6-886e-39c25624bcf7)

How can I delete the pink color space in the div tag?
As shown in Figure. The space is after the Home text.

My Code:-
                
![image](https://github.com/tailwindlabs/tailwindcss/assets/122990718/a4f6ebe3-357c-4e79-a1ac-d5cff91238cc)

Thanks
Minal P

**Comment by wongjn** [comment, marked as answer]
You'd want to "shrink-wrap" the flex parent by applying `width: max-content` (or equivalent) via `w-max`:

```html
<div class="flex items-center w-max">
```
---

As an aside, [please consider not upload images of code/data/errors.](https://meta.stackoverflow.com/q/285551/19655631)
